common/src/View/EntityAttributeGridTable.cpp

```diff
@@ -584,6 +586,22 @@ namespace TrenchBroom {
             return valueset;
         }
 
+        StringSet EntityAttributeGridTable::allSortedClassnames(MapDocumentSPtr document) {
+            // Start with classnames in use in the map
+            StringSet valueset = allSortedValuesForAttributeNames(document, StringList{ Model::AttributeNames::Classname });
+
+            // Also add keys from all loaded entity definitions
+            for (const auto& group : document->entityDefinitionManager().groups()) {
```

Comment:
Just use `EntityDefinitionManager::definitions`, which returns a reference to a list containing all definitions.